完善/更新部分文档 (#1538)
cocos-creator/creator-docs: en/scripting/setup.md

## Editing scripts

> Users can choose their favorite text-editing tools (such as: **Vim**, **Sublime Text**, **Web Storm**, **VSCode**...) for script editing according to their own needs, please review [Settings](#defaultscripteditingtoolconfiguration) to change.

Reviewer: ```suggestion
Choose a favorite text-editing tool (such as: **Vim**, **Sublime Text**, **Web Storm**, **VSCode**...) for script editing, please review [Settings](#defaultscripteditingtoolconfiguration) to change.
```

By double-clicking the script resource, the script editor directly opens to allow for editing. When the script is edited and saved, then returned to the editor, Cocos Creator will automatically detect the changes to the script and compile it quickly.

Before writing code, please read [Basics of Scripting](basic.md) documentation to learn more about scripts.

## Add a script to a scene node

Adding a script to a scene node is actually adding a script component to that node. Select the scene node to which you wish to add a script in the **Hierarchy** panel, at which point the properties of that node will be displayed in the **Inspector** panel. Adding a script component includes the following two ways:

1. Drag and drop the script from **Assets** panel directly into the **Inspector** panel.

    ![add scriptcomponent](setup/add-script-component.png)

> 2. Click the **Add Component** button at the bottom of the **Inspector** panel and select **Custom script -> SayHello** to add the script component just created. Or, add it by searching for **SayHello** in the Add Component search box, provided that the **Add components using popups** feature in **Preferences -> Laboratory** is enabled.

Reviewer: ```suggestion
2. Click the **Add Component** button at the bottom of the **Inspector** panel and select **Custom script -> SayHello** to add the script component just created. It is also possible to add it by searching for **SayHello** in the **Add Component** search box, provided that the **Add components using popups** feature in **Preferences -> Laboratory** is enabled.
```
